manifests/get_from_github.pp

```diff
@@ -1,17 +1,26 @@
-# private
-define collectd::get_from_github(
-  $localfolder,
-  $source
+define collectd::get_from_github (
+  $localfolder = undef,
+  $source      = undef,
+  $ensure      = present,
+  $revision    = 'master',
 ) {
-  if(!defined(Vcsrepo[$localfolder])){
-    collectd::check_and_install_package { $title:
-      package_name => 'git'
-    } ->
+  validate_string($source)
+  validate_string($revision)
```

Comment:
looks like Puppet Enterprise stopped shipping stdlib as of 3.7.  if that is true then maybe add requirement to readme.